woocommerce/wc-admin
# Reduce order table net revenue by refund total

Fixes #739 

Updates the net revenue used for order table data and summary data to reflect refund amounts.

### Screenshots

#### Before
<img width="562" alt="screen shot 2018-12-04 at 12 21 17 pm" src="https://user-images.githubusercontent.com/10561050/49418853-22d50480-f7bf-11e8-9d2f-da31c9b93f9e.png">

#### After
<img width="737" alt="screen shot 2018-12-04 at 12 20 45 pm" src="https://user-images.githubusercontent.com/10561050/49418854-25cff500-f7bf-11e8-92a4-9795e433d44c.png">

### Detailed test instructions:

1. Create an order with any amount of refunds.
2.  You may also need to manually click "Update" once more on this refund if you're processing from the dashboard.
3.  Visit the orders page `wp-admin/admin.php?page=wc-admin#/analytics/orders`.
4.  Check that the correct amount is reflected in both the table and the summary beneath the table.
5.  Run phpunit tests and make sure all pass.
